packages/components/src/internal/components/samples/SampleCreationTypeModal.tsx

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import { Button, FormControl, Modal } from "react-bootstrap";
+import { MAX_EDITABLE_GRID_ROWS } from "../../../index";
+import {
+    SampleCreationTypeOption
+} from "./SampleCreationTypeOption";
+import { SampleCreationType, SampleCreationTypeModel } from "./models";
+
+
+interface Props {
+    show: boolean;
+    options: Array<SampleCreationTypeModel>,
+    parentCount: number;
+    showIcons: boolean;
+    onCancel: () => void;
+    onSubmit: (creationType: SampleCreationType, numPerParent?: number) => void;
+}
+
+interface State {
+    numPerParent: number;
+    creationType: SampleCreationType;
+    submitting: boolean;
+}
+
+export class SampleCreationTypeModal extends React.PureComponent<Props, State> {
+
+    constructor(props: Props) {
+        super(props);
+
+        props.options.find(option => option.selected);
```

Comment:
I assume this can be removed. The `find` doesn't end up in a variable or get used.